File: mycollab-web/src/main/java/com/esofthead/mycollab/module/file/events/CloudDriveOAuthCallbackEvent.java
package com.esofthead.mycollab.module.file.events;

import com.esofthead.mycollab.eventmanager.ApplicationEvent;

public class CloudDriveOAuthCallbackEvent {

	public static class ReceiveCloudDriveInfo extends ApplicationEvent {
		private static final long serialVersionUID = 1L;

		public ReceiveCloudDriveInfo(Object source, Object data) {
			super(source, data);
		}
	}
}

File: mycollab-services/src/main/scala/com/esofthead/mycollab/module/ecm/esb/CloudDriveOAuthCallbackEvent.scala
package com.esofthead.mycollab.module.ecm.esb

import com.esofthead.mycollab.eventmanager.ApplicationEvent
import com.esofthead.mycollab.module.file.CloudDriveInfo

/**
  *
  * @author MyCollab Ltd.
  * @since 1.0
  *
  */
object CloudDriveOAuthCallbackEvent {

  @SerialVersionUID(1L)
  class ReceiveCloudDriveInfo(source: AnyRef, data: CloudDriveInfo) extends ApplicationEvent(source, data) {}

}